package.json

```diff
@@ -29,10 +29,18 @@
   "devDependencies": {
     "@commitlint/cli": "^19.0.3",
     "@commitlint/config-conventional": "^19.0.3",
+    "@types/leaflet": "^1.9.9",
```

Comment:
**issue (blocking):** The leaflet types, react , react-dom packages etc shouldn't be installed here. They should be installed in the frontend package. Kindly review and remove any unrequired package